package.json

```diff
@@ -94,7 +94,7 @@
     },
     "dependencies": {
         "@js-joda/core": "^3.2.0",
-        "catbuffer-typescript": "1.0.0",
+        "catbuffer-typescript": "^1.0.1-alpha-202111091444",
```

Comment:
why are we depending on alpha libs?

Reply:
This PR goes together with https://github.com/symbol/catbuffer-generators/pull/166. Once that pr (and the revokable mosaic support) is merged we can release it and upgrade the versions here.